keyboards/mechmini/v2/keymaps/wsturgiss/keymap.c

```diff
@@ -0,0 +1,107 @@
+#include QMK_KEYBOARD_H
+#include "../../../../../quantum/rgblight_list.h"
```

Comment:
Per @drashna's review, this line should be deleted. `#include QMK_KEYBOARD_H` automatically includes this file if `RGBLIGHT_ENABLE = yes`.

```suggestion
```

Reply:
done